### PR TITLE
Add OriginTrail Parachain to SS58 Registry

### PR DESCRIPTION
**Pull request by kotlarmilos**
Add OriginTrail Parachain (https://origintrail.io/) ss58 address prefix:

- update `crypto.rs`
- update `ss58-registry.json`

### CODE REVIEW

**Review by BenWhiteJam — APPROVED** [review]
fine from BD side.

- [x] contact
- [x] testnet
- [x] code repo
- [x] support
- [x] community

File: ss58-registry.json [hunk 1]
```diff
@@ -505,6 +505,15 @@
 			"standardAccount": "*25519",
 			"website": "https://manta.network"
 		},
+		{
+			"prefix": 101,
+			"network": "origintrail-parachain",
+			"displayName": "OriginTrail Parachain",
+			"symbols": ["TRAC"],
+			"decimals": [18],
+			"standardAccount": "ethereum",
```
Comment (bkchr) [inline ss58-registry.json:514]:
```suggestion
			"standardAccount": "secp256k1",
```

**Review by bkchr — APPROVED** [review]

**Comment by bkchr** [comment]
bot merge

**Comment by bkchr** [comment]
@kotlarmilos could you please also merge master, ty.

**Pull request merged**